roles/ideascube/tasks/branding.yml

```diff
@@ -0,0 +1,7 @@
+---
+- include_vars: group_vars/{{ ansible_architecture }}.yml
+  tags: ['master', 'custom']
+
```

Comment:
You have to create the `/usr/share/ideascube/branding/` directory first. cf https://github.com/ideascube/ansiblecube/blob/oneUpdateFile/roles/system/tasks/main.yml#L284 for example.